index.html

```diff
@@ -0,0 +1,193 @@
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="section_2">
+      <div class="tonic_image_1">
+        <img src="images/post_story2.png" alt="Tonic image">
+      </div>
+      <div>
+      <div class="tonic_div">
+        <h2>Multi-Post
+          Stories</h2>
+        <ul>
+          <li class="tonic_title">CANOPY</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">Back End Dev</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">2015</li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about_section">
+      <h2 class="about_title">About Myself
+      </h2>
+        <p>Hello I'm software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <div class="about_social_media1">
+        <h2 class="connect">LET'S CONNECT</h2>
+        <div class="about_social">
+          <a href=""><img src="images/Vector5.png" alt=""></a>
+          <a><img src="images/Linkedin icon.png" alt=""></a>
+          <a><img src="images/Vector4.png" alt=""></a>
+          <a><img src="images/Vector2.png" alt=""></a>
+          <a><img src="images/Vector3.jpg" alt=""></a>
+        </div>
+        <div class="view_button">
+          <button>Get my resume</button>
+        </div>
+        <div class="language">
+        <h2>Languages</h2>
+        <img src="images/Union.png" alt="drop down image">
+        </div>
+        <div class="languages">
+          <button><img src="images/js.png" alt=""> <span>JavaScript</span></button>
+          <button><img src="images/html.png" alt=""><span>HTML</span></button>
+          <button><img src="images/css.png" alt=""> <span>CSS</span></button>
+        </div>
+        <div class="language">
+          <h2>Frameworks</h2>
+          <img src="images/Union_right.png" alt="drop down image">
+          </div>
+          <img src="images/Separator Bottom.png" alt="horizontal line">
+          <div class="language">
+            <h2>Skills</h2>
+            <img src="images/Union_right.png" alt="drop down image">
+            </div>
+            <img class="hr_line" src="images/Separator Bottom.png" alt="horizontal line">
```

Comment:
- [ ] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li, and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li. This is a project requirement that ensures your code is well structured. 🤝